db/migrate/20190808192157_create_help_texts.rb

```diff
@@ -0,0 +1,19 @@
+class CreateHelpTexts < ActiveRecord::Migration[5.1]
+  def up
+    create_table :help_texts do |t|
```

Comment:
Why add a new table for help texts and not add columns to the service provider table?

Reply:
Great question... With the use of multiple 'sections' it would be either three separate columns (`sign_in_help_text`, `sign_up_help-text`, `forgot_password_help_text`) or one really complicated column (`help_text: { sign_in: { en: '', es: '', fr:'' }, { sign_up: { en: '', es: '', fr:'' }, { forgot_password: { en: '', es: '', fr:'' } }, in the service_provider table. 

That, with the potential for even more sections if one day users wanted to put help text on another page. Felt like enough potential repetition within the (already huge) service provider model to just make help text its own model.

Reply:
Also, I'm open to the argument that not much is being done in terms of logic specific to the help text model at the moment... But even given that, I decided it was just a lot to put in the service provider table.